contracts/StreamManager.sol

```diff
@@ -196,25 +168,23 @@ contract StreamManager is IStreamManager, ReentrancyGuard {
     ///@dev payee can claim tokens which is proportional to elapsed time (exactly seconds).
     function claim()
         onlyPayee
-        notTerminatedInCliffPeriod
         onlyAfterCliffPeriod
         nonReentrant
         external
     {
         uint256 claimedAt = block.timestamp;
         uint256 terminatedAt = streamInstances[msg.sender].terminatedAt;
-        uint256 lastClaimedAt = streamInstances[msg.sender].lastClaimedAt;
         address token = streamInstances[msg.sender].token;
         uint256 terminationPeriod = streamInstances[msg.sender].terminationPeriod;
         bool isTerminated = streamInstances[msg.sender].isTerminated;
         uint256 claimableAmount;
 
-        if (!isTerminated || isTerminated && claimedAt <= terminatedAt + terminationPeriod) {
+        if (!isTerminated || isTerminated && claimedAt < terminatedAt + terminationPeriod) {
```

Comment:
combining || and && should be done like that a || (b && c) of (a || b) && c

Reply:
https://github.com/holdex/payment-streaming/pull/88#discussion_r1269623174

Reply:
Yeah